Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -433,20 +434,51 @@ public override void Update(float frameTime)
 
                 if (flammable.FireStacks > 0)
                 {
-                    var air = _atmosphereSystem.GetContainingMixture(uid);
+                    var air = _atmosphereSystem.GetContainingMixture(uid, false, true);
```

Comment:
Consider not exciting here, and then moving the excite to after the following check is complete. I believe exciting a gas mixture incurs somewhat expensive processing.

Reply:
Looking at it, I think if we can change it in AtmosExposed we shouldn't need to excite here since Flammable can ensure AtmosExposed (though really flammable entities should just have the AtmosExposedComponent prototypes, but ECS moment whatcha gonna do).